src/Native/gen-buildsys-clang.sh

```diff
@@ -46,11 +46,11 @@ OS=`uname`
 # Locate llvm
 # This can be a little complicated, because the common use-case of Ubuntu with
 # llvm-3.5 installed uses a rather unusual llvm installation with the version
-# number postfixed (i.e. llvm-ar-3.5), so we check for that first.
+# number postfixed (i.e. llvm-3.5), so we check for that first.
 # On FreeBSD the version number is appended without point and dash (i.e.
-# llvm-ar35).
+# llvm-35).
 # Additionally, OSX doesn't use the llvm- prefix.
-if [ $OS = "Linux" -o $OS = "FreeBSD" -o $OS = "OpenBSD" -o $OS = "NetBSD" ]; then
+if [ $OS = "Linux" -o $OS = "DragonFly" -o $OS = "FreeBSD" -o $OS = "NetBSD" -o $OS = "OpenBSD" ]; then
   llvm_prefix="llvm-"
 elif [ $OS = "Darwin" ]; then
   llvm_prefix=""
```

Comment:
Just found a third issue: Are we even using locate_llvm_exec or llvm_prefix anymore?